ir/src/symbol_table.rs

```diff
@@ -20,6 +20,9 @@ pub(super) enum IdentifierType {
     PeriodicColumn(usize, usize),
     /// an identifier for an integrity variable, containing its name and value
     IntegrityVariable(Variable),
+    /// an identifier for random value, containing its index in the random values array and its
+    /// length if this value is an array. For non-array random values second parameter is always 1.
+    RandomValue(usize, usize),
```

Comment:
nit: If this can represent an array, should this be called RandomValues instead?

Reply:
The problem is that we have a main array of random values (`rand: [a, b, c, ... ]`), so calling it `RandomValues` can be confusing, it will be unclear whether we mean the main array or its element, which is represented by an array. I agree with you that it would be better to rename it somehow, but I don't know yet which name would be better. Perhaps `RandomValuesElement', but this name also does not reflect that this element can be an array.

Reply:
It could be `RandomValuesBinding` again, to be somewhat consistent with the AST?

Reply:
What about `RandomValuesGroup`?